src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -64,6 +64,9 @@
     public static final byte[] EMPTY_BYTE_ARRAY = new byte[0];
     /** Copied from org.apache.commons.lang3.ArrayUtils. */
     public static final double[] EMPTY_DOUBLE_ARRAY = new double[0];
+    /** Pseudo URL prefix pattern for loading from the class path. */
+    public static final Pattern CLASSPATH_URL_PREFIX_PATTERN =
+        Pattern.compile("classpath:", Pattern.LITERAL);
```

Comment:
should be `"^classpath:"` to strictly match prefix

Reply:
Done.